src/DurableFunctions.TypedInterfaces/Example/SimpleOrchestration.cs

```diff
@@ -18,7 +18,7 @@ public class SimpleOrchestration
         [FunctionName("SimpleOrchestrationHttp")]
         public static async Task<IActionResult> HttpStart_SimpleOrchestration(
             [HttpTrigger(AuthorizationLevel.Anonymous, "get")] HttpRequest req,
-            [DurableClient] ITypedDurableClient client,
+            [DurableClient] IDurableClient client,
```

Comment:
Does this work? I'd expect it to create a compilation failure on line 24.  Maybe this can be removed from the PR for now?

Reply:
You're right, it does cause compilation failure. I'm going to revert this and we can determine what the correct solution is for the warning later.

Reply:
I'm also fine if we just suppress this warning.

Reply:
Suppressed